spot_rl_experiments/spot_rl/real_policy.py

```diff
@@ -32,68 +47,133 @@ def __init__(
         action_space,
         device,
         policy_class=PointNavBaselinePolicy,
+        is_hab3_policy=False,
     ):
```

Comment:
Can we get some class-level, module-level docstrings? It is hard to understand what is the purpose of this file.

Reply:
Posted this on slack channel